AnnoyancesFilter/sections/popups.txt

```diff
@@ -101,6 +101,7 @@ czasnabuty.pl###topLayerBackDrop
 czasnabuty.pl###toplayer_newsletter_popup
 sowafinansowa.pl##.popover#last_buy_pop
 tproger.ru###float-otus
+pornxt.com#$##playerOverlay { z-index: -999 !important; }
```

Comment:
why not element hiding?

Reply:
somehow it didn't work. but now `pornxt.com###playerOverlay` works just fine so I committed it